go.mod

```diff
@@ -9,7 +11,7 @@ replace (
 
 require (
 	github.com/XSAM/otelsql v0.26.0
-	github.com/a-h/templ v0.2.432
+	github.com/a-h/templ v0.3.833
```

Comment:
make sure to regenerate the templ files > v0.2.747
the versions don't match

Reply:
Looks like mine was very outdated :) ty